src/Tools/dotnet-monitor/Experimental/ExperimentalFlags.cs

```diff
@@ -17,16 +17,27 @@ internal class ExperimentalFlags : IExperimentalFlags
         private const string ExperimentalPrefix = ToolIdentifiers.StandardPrefix + "Experimental_";
```

Comment:
Please revert the changes to this file; the experimental feature flags for these two features were removed in favor of using configuration options.